src/server/cmd/pachd/main.go

```diff
@@ -140,6 +133,7 @@ func do(appEnvObj interface{}) error {
 			pfsclient.RegisterBlockAPIServer(s, blockAPIServer)
 			ppsclient.RegisterAPIServer(s, ppsAPIServer)
 			ppsserver.RegisterInternalJobAPIServer(s, ppsAPIServer)
+			persist.RegisterAPIServer(s, rethinkAPIServer)
```

Comment:
Diff is kinda weird but this is the real change here right, just having the rethink server registered.